doc/md/guides/overview/modern-database-ci-cd.mdx

```diff
@@ -0,0 +1,233 @@
+```
+git add -A .
+git commit -m "Add email index"
+git push origin add-email-index
+```
+
+Your CI pipeline should be configured to run the following steps whenever a new commit is pushed to the branch.
+Teams commonly configure these steps to run only when a change is made to the desired state of the database or to
+the migration directory.
+
+```bash
+# login to Atlas to report migration analysis report
+atlas login --token $ATLAS_TOKEN
+
+# populate a JSON object with metadata about the Git repo, branch, commit and URL (usually the PR URL)
+ATLAS_CONTEXT=$(cat <<EOF
+{
+    "repo": "$CI_PROJECT_PATH",
+    "branch": "$CI_COMMIT_REF_NAME",
+    "commit": "$CI_COMMIT_SHA",
+    "url": "$URL"
+}
+EOF
+)
+
+# Run analysis against the latest revision of the database.
+atlas migrate lint --base atlas://project-id --context $ATLAS_CONTEXT
+```
+
+This way, whenever a change is made to the desired state of the database, the CI pipeline will automatically
+simulate and analyze the changes and verify that they are valid and safe.  Atlas will generate a detailed
+analysis report, an example of which you can see [here](https://gh.atlasgo.cloud/dirs/4294967383/ci-runs/8589934684).
+
+:::info Further reading
+
+It is possible to push your changes to Atlas Cloud directly with the CLI
+your CI pipeline, but official integrations are also available:
+
+* The `atlas-action` [GitHub Action](/integrations/github-actions#arigaatlas-action-ci)
+* The [GitLab CI](/guides/ci-platforms/gitlab) Guide (includes instructions for running `migrate lint` in pipelines)
+
+:::
+
+### 5. Merge your changes to push the latest revision of the database
+
+Once CI is green and the change is approved, the changes should be merged into the main branch.
+This merge should trigger another workflow, which will push the most recent version of the migration
+directory to [Atlas Cloud](https://atlasgo.cloud). You can think of this step as similar to pushing
+a Docker image to a container registry when code updates are merged into the main branch.
+
+This is typically done by running the following steps in your workflow:
+
+```bash
+# login to Atlas to push the migration directory
+atlas login --token $ATLAS_TOKEN
+
+# push the migration directory to Atlas Cloud
+atlas migrate push project-id:$GIT_COMMIT
+atlas migrate push project-id:latest
+```
+
+Once the migration directory is pushed to Atlas Cloud, you can view a visual representation of the
+database schema and its revision history in the Atlas Cloud UI, you can see an example of this
+[here](https://gh.atlasgo.cloud/dirs/4294967383).
+
+:::info Further reading
+
+It is possible to use the Atlas CLI directly in your CI pipeline, but official integrations are also available:
+
+* The `atlas-sync-action` [GitHub Action](/integrations/github-actions#arigaatlas-sync-action)
+* The [GitLab CI](/guides/ci-platforms/gitlab) guide, includes instructions for running `migrate push` in pipelines.
+
+:::
+
```

Comment:
This block is doubled. It is present in the previous section as well. By design?

Reply:
Yes, it links to a different action